### ISSUE TITLE
Have the bot open PRs

### ISSUE THREAD

**Issue by hectorsector** [githubtraining/write-a-ll-course#14]
Originally brought up on #11, having the bot open a PR would have a few pros:
- We could skip the welcome issue 
- Declare titles of the pull request so we know how to reference them

And a few cons:
- The bot cannot approve its own PR
- We have to remove and reapply branch protections, or have learners approve (which is an extra step)
- We have to set up the branches on the template repo how we expect them. When the branches are all addressing a single file, it takes a lot of care to ensure there are no merge conflicts

/cc @brianamarie who brought it up

**Comment by hectorsector** [comment]
@brianamarie I'm assigning this to you following our discussion in 1:1. I'm on board for having the bot open PRs, but the template repo needs to then include separate branches with which to open the PRs, and those PRs should not produce merge conflicts. One way I was able to do this in the CI course is to separate changed hunk with comments/.

**Comment by brianamarie** [comment]
@hectorsector This is on my list today. 👍

One flow question - right now, we have the user create a title in the beginning, but wait to create a description and tagline until the end. Can you help me understand why we don't have the user tackle all three of those things at the beginning?

**Comment by brianamarie** [comment]
@hectorsector I've created an alternative template repository. I didn't want to overwrite the existing one, in case this breaks stuff. :bowtie: https://github.com/githubtraining/write-a-ll-course-template-2

It still has a `metadata` branch, so it _should_ work with the existing course. The other branches are named `title` for PR 1 (steps 1-2), `repo-setup` for PR 2 (steps 3-6), `write-steps` for PR 3 (steps 7-11), and `publishing` for PR 4 (steps 12-14). 

I should note that in the template, I included the tagline and description information in the first chunk. I think we should consider moving step 12 to be after step 2.

**Comment by hectorsector** [comment]
> right now, we have the user create a title in the beginning, but wait to create a description and tagline until the end. Can you help me understand why we don't have the user tackle all three of those things at the beginning?

A pretty annoying pain point for me is trying to come up with a handful of descriptions before I get down to any "fun". Besides, in the beginning it's difficult to have a clear description of the course ready. For that reason, I decided to do it later. However, since we're making use of suggested changes it might not be a big deal to do it all at once.

**Comment by hectorsector** [comment]
@brianamarie there are no changes between `master` and any of the new branches.
